css/logo-nav.css

```diff
@@ -680,6 +680,126 @@ body {
     background-color: white;
 }
 
+@media(max-width:509px) {
```

Comment:
There is one 509px here,  the other is indeed in legacy code, I would suggest using 480 here, and fixing the already existing 509 later.

Reply:
ya, it looks like the original author is creating his own break point. I would recommend using what's default to this version of bootstrap

Reply:
Yeah, it looks like this was a hack to make things 'look right' in a very specific context, this was never meant to be production code to my understanding